sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -370,9 +344,7 @@ export class ServiceBusManagementClient extends ServiceClient {
 }
 
 // @public
-export interface ServiceBusManagementClientOptions {
-    proxySettings?: ProxySettings;
-    userAgentOptions?: UserAgentOptions;
+export interface ServiceBusManagementClientOptions extends PipelineOptions {
```

Comment:
Can we use the PipelineOptions directly and not have ServiceBusManagementClientOptions at all?

Reply:
Can do!

Reply:
Done!